integr8ly/cloud-resource-operator
# do not specify cache instances when listing availabile instance types

verification:
- install a redis aws resource in stockholm
- override the strategy configmap to override it to cache.t2.micro
- ensure it's not overridden and the redis cr is in a successful
  state
